styles/app.scss

```diff
@@ -15,10 +15,10 @@
   @font-face {
     font-family: "Lato";
     font-style: normal;
```

Comment:
If we're changing the font-weight  from 400 -> 700 I'm wondering if we should also update the font-style to `bold`.  If we want `bold` to the default for this style I wonder if that's something we can achieve through the tailwind config?  I bring it up because I can see a scenario in the future where a developer is trying to style text like a design and can't figure out why their explicit `normal` font class declaration keeps coming out bold.

Reply:
In a nutshell should be leave both the 400 and 700 weight fonts declared but set the 700 as default for text.

Reply:
Ah I see what you mean! [font-style](https://developer.mozilla.org/fr/docs/Web/CSS/font-style) is for normal/italic/oblique. Usually, we should set a font-face to match the font file.